src/eduid_idp/mischttp.py

```diff
@@ -20,7 +20,10 @@
 import cherrypy
 import pkg_resources
 
-from urlparse import parse_qs
+try:
+    from urlparse import parse_qs
```

Comment:
Shouldnt it be "from urlparse import parse_qs as urlparse"?

Reply:
excellent, fixed